includes/wikia/services/UserService.class.php

```diff
@@ -19,31 +23,35 @@ public static function getNameFromUrl($url) {
 
 	/**
 	 * get main page for current user respecting user preferences
+	 *
 	 * @param User $user
+	 *
 	 * @return Title
 	 */
-	public static function getMainPage(User $user) {
+	public static function getMainPage( User $user ) {
 		$title = Title::newMainPage();
 
-		if( $user->isLoggedIn() ) {
-			$value = $user->getGlobalPreference(UserPreferencesV2::LANDING_PAGE_PROP_NAME);
-			switch($value) {
+		if ( $user->isLoggedIn() ) {
+			$value = $user->getGlobalPreference( UserPreferencesV2::LANDING_PAGE_PROP_NAME );
+			switch ( $value ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)